new icon: laminas (original, original-wordmark, line, line-wordmark)

hammad-521 opened the pull request:
## Double check these details before you open a PR

<!-- Tick the checkboxes to ensure you've done everything correctly -->
- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

<!-- Refer to the [contributing](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) guidelines for more details. -->

**This PR closes #2234**
<!-- List issues that this PR would close above. Ex: This PR closes #1, #2, #3. -->

## Link to prove your SVG is correct and up-to-date.
<!-- Link to an official page/wiki goes here. Anything that proves your SVGs are the correct ones. -->
https://getlaminas.org/

canaleal commented on icons/laminas/laminas-line.svg:
SVG should be scaled 128 so it can reach the edges of the viewbox. 
The strokes in the SVG should also be converted into paths.

![image](https://github.com/user-attachments/assets/844dee67-38a1-4b39-ae52-41336ea3de3e)

canaleal commented on icons/laminas/laminas-line-wordmark.svg:
Convert all strokes in this SVG into paths.

Icomoon, the tool we use to convert SVGs into font icons, requires paths to function correctly. If an SVG doesn't contain paths, it won't be converted properly and will appear empty.

![image](https://github.com/user-attachments/assets/ccc800ab-5b3d-4fb5-8ac6-4c46262d1664)

canaleal commented on devicon.json:
```diff
@@ -5489,6 +5489,42 @@
+                "line-wordmark"
+            ],
+            "font": [
+                "original"
```
Add, `original-wordmark`, `line`, and `line-wordmark` here as all 3 can be converted into font versions.

canaleal commented on devicon.json:
```diff
@@ -5489,6 +5489,42 @@
+            ]
+        },
+        "color": "#013755",
+        "aliases": [
```
Avoid using aliases for existing SVG versions. For example, original and original-wordmark are distinct icons and already exist as SVGs.

Instead, use aliases for versions that aren't included in the `version.svg` or `version.font`. For instance, the plain version of an icon doesn't exist as an SVG, but since original icons are simple enough (They only have one fill color and one path) they can represent the plain versions of the icons. 

I recommend only including this alias:

    Base: original
    Alias: plain
    
    Base: original-wordmark
    Alias: plain-wordmark

canaleal reviewed (changes requested):
Overall excellent work, just a few minor comments that have to be resolved.

hammad-521 commented:
@canaleal 
Thank you for your thorough review. I've addressed all the points you raised:

1. icons/laminas/laminas-line.svg:
   - Scaled the SVG to 128 to reach the edges of the viewbox.

2. icons/laminas/laminas-line-wordmark.svg:
   - Converted all strokes to paths.

3. devicon.json:
   - Added 'original-wordmark', 'line', and 'line-wordmark' to the font versions.
   - Updated the aliases section as suggested:
     ```
     "aliases": [
       {
         "base": "original",
         "alias": "plain"
       },
       {
         "base": "original-wordmark",
         "alias": "plain-wordmark"
       }
     ]
     ```

I've committed these changes and pushed them to the repository. The pull request now shows 2 commits. Could you please review the changes again? Let me know if any further modifications are needed.

canaleal reviewed (dismissed):
Excellent work 💯
Just added another commit to fix some line artifacts on icomoon for the line SVGs.

canaleal commented:
No artifacts and all SVGs are good. Great work:

![image](https://github.com/user-attachments/assets/b84e48ff-ba27-468d-95e5-9a4fc93bfdd6)
